src/platform/forms/tests/save-in-progress/RoutedSavableReviewPage.unit.spec.jsx

```diff
@@ -223,22 +225,28 @@ describe('Schemaform save in progress: RoutedSavableReviewPage', () => {
       },
     };
 
-    const tree = shallow(
-      <RoutedSavableReviewPage
-        router={router}
-        setData={setData}
-        openChapters={[]}
-        form={form}
-        user={user}
-        onSubmit={onSubmit}
-        setEditMode={f => f}
-        setPrivacyAgreement={f => f}
-        formConfig={route.formConfig}
-        pageList={route.pageList}
-        path={route.path}
-        location={location}
-      />,
-    );
+    let tree;
+    beforeEach(() => {
```

Comment:
I put this into a `beforeEach()` block for now instead of trying to significantly refactor these tests.